deploy/kubernetes/rbac.yaml

```diff
@@ -21,9 +21,10 @@ apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: external-provisioner-runner
 rules:
-  - apiGroups: [""]
-    resources: ["secrets"]
-    verbs: ["get", "list"]
+  # Some provisioners may need to uncomment the following rule.
```

Comment:
Maybe something like "The following rule should be uncommented for plugins that require secrets for provisioning"

Reply:
Comment updated.